src/docs/markdown/caddyfile/directives/log.md

```diff
@@ -13,6 +13,10 @@ $(function() {
 
 Enables and configures HTTP request logging (also known as access logs).
 
+<aside class="tip">
+	If you're looking to configure Caddy's runtime logs, you're looking for the <a href="/docs/caddyfile/options#log"><code>log</code> global option</a> instead.
```

Comment:
Totally a nit, but does this make sense? I dunno what else to call them. Runtime is fine. I usually have called them Process logs. Or is there another word?

```suggestion
	If you're looking to configure Caddy's process logs, you're looking for the <a href="/docs/caddyfile/options#log"><code>log</code> global option</a> instead.
```

Reply:
Both are essentially equivalent I guess. You could say "the Caddy runtime" or "the Caddy process", I think "runtime" might make more sense when talking about the lifecycle of Caddy, because a config != process' lifetime